### PR TITLE
#165378746 User can get a list of suggested users to follow

### PR DESCRIPTION
**Pull request by sojida**
#### Description
Currently, when a user has activities on the application, we can use those activities to suggest follows for the user.

#### Type of change

- [x] New Feature(create route to get user suggestions)

#### How Has This Been Tested?

- [x] End to End testing

#### Checklist:
- [x] Write test cases
- [x] Create a route to get suggested article

#### PT-ID

[#165378746](https://www.pivotaltracker.com/story/show/165378746)

#### Screenshots
Screenshot 2019-04-16 at 4 18 23 PM

#### Questions:
N/A

### CODE REVIEW

File: server/helpers/aurhors-id-helper.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const getAuthorsIdFromArticle = (req, obj) => {
+  const allUserIds = obj.map(item => item.Article.author.id);
+
+  const uniqueIds = [...new Set(allUserIds)];
+
+  const removeUserOwnId = uniqueIds.filter(
```
Comment (rotimi-babalola) [inline server/helpers/aurhors-id-helper.js]: Give this is a better name. This sounds like the name of a function

File: server/routes/profile.routes.js [hunk 2]
```diff
@@ -113,4 +113,14 @@ router.patch(
   profileController.patchProfile
 );
 
+router.get(
+  '/suggested/researchers',
+  tokenValidator.verifyToken,
+  profileMiddleware.getAuthorOFArticleUserLiked,
```
Comment (rotimi-babalola) [inline server/routes/profile.routes.js]: Should be `Of`

File: server/helpers/aurhors-id-helper.js (hunk 1 above)
Comment (sojida) [reply server/helpers/aurhors-id-helper.js]: 👍🏾

File: server/routes/profile.routes.js (hunk 2 above)
Comment (sojida) [reply server/routes/profile.routes.js]: 👍🏾I' ll fix this

File: server/helpers/aurhors-id-helper.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const getAuthorsIdFromArticle = (req, obj) => {
+  const allUserIds = obj.map(item => item.Article.author.id);
+
+  const uniqueIds = [...new Set(allUserIds)];
+
+  const ids = uniqueIds.filter(item => item !== req.user.userObj.id);
```
Comment (rotimi-babalola) [inline server/helpers/aurhors-id-helper.js]: You could just return `uniqueIds.filter(...)` directly

Comment (rotimi-babalola) [reply server/helpers/aurhors-id-helper.js]: And leave a comment explaining what it's doing

**Pull request merged**